### ISSUE TITLE
Issue with ng-bootstrap 14.1.0/Angular 15.2.8 tooltip

### ISSUE THREAD

**Issue by hairbo** [ng-bootstrap/ng-bootstrap#4494]
### Bug description:
If I encapsulate an HTML element which calls `ngbTooltip` inside an Angular component (where, for example, I might pass in the tooltip content as an argument), when I mouse over that element, the content _after_ the tooltip-enabled element wraps to the next line. I get the same behavior with several different inline elements: span, button, a. 

### Link to minimally-working StackBlitz that reproduces the issue:
https://stackblitz.com/edit/angular-fagut4

Here's a Stack Overflow post I just made that has a little more detail, if it's helpful (but I would hope everything you need is in the Stackblitz example--there's not much to it):
https://stackoverflow.com/questions/76095741/problem-with-ng-bootstrap-14-tooltip-in-angular-15

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 15.2.8

ng-bootstrap: 14.1.0

Bootstrap: 5.2.3

@popperjs/core: 2.11.6

**Comment by DDtMM** [comment]
I answered your SO post and provided a couple of work arounds, the easiest being adding the container="body" attribute.   Also, I always just default to using container="body" because of issues like this that I experienced years ago.  I don't understand why that isn't the default, but I've never run into it being a problem.

I don't think this is an ngBootstrap issue, and instead either a popperjs or twbs issue.
